project/newrelic_webmon/newrelic.ini

```diff
@@ -0,0 +1,195 @@
+# ---------------------------------------------------------------------------
```

Comment:
Less moving pieces. In reality we aren't really changing much of the configuration. I also stopped giving different app names to the Celery worker and the Gunicorn instances because it leads to a lot of weird repeated information on the New Relic dashboard.